### PR TITLE
16-04-21 Blogpost

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: _posts/2021-04-16-status-update.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "2021-04-16: IAC & DevX Team Status Update"
+author: david22swan
+categories:
+  - team
+  - status
+tags:
```
Comment (daianamezdrea) [inline _posts/2021-04-16-status-update.md:8]: Please add some tags here

File: _posts/2021-04-16-status-update.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+I am happy to announce the end of our terminology work, as of yesterday all *master* branches have been removed from our supported modules and tools, and with the previous work done to remove any mentions I am happy to declare this piece of work closed.
+
+A big shout out to [David Swan][DavidSwan] (i.e. Me!) for heading up this work.
+
+## Intern Update
+
+Disha’s attended her 1st Puppet Camp this week! Check out her blog of what she is up to this week [here](https://puppetlabs.github.io/iac/lifeofinternatpuppet/post_31.html)
+
+## Other Work
```
Comment (DavidS) [inline _posts/2021-04-16-status-update.md:65]: If there's no other work, please remove the heading

Comment (david22swan) [reply _posts/2021-04-16-status-update.md]: done

File: _posts/2021-04-16-status-update.md (hunk 1 above)
Comment (david22swan) [reply _posts/2021-04-16-status-update.md:8]: Done

**Review by DavidS — APPROVED** [review]

**Pull request merged**